templates/index.html.hbs

```diff
@@ -32,6 +32,31 @@
       Author: BootstrapMade.com
       License: https://bootstrapmade.com/license/
     ======================================================= -->
+
+    <script>
+      function requestBackend() {
+        var email = document.getElementById("email-subscribe").value;
+
+        var http = new XMLHttpRequest();
+        var url = 'http://127.0.0.1:8000/newsletter';
```

Comment:
You shoudln't use the domain name and use absolute path insead `/newsletter`

Reply:
done